app/src/main/kotlin/com/wire/android/ui/home/conversations/media/ConversationMediaScreen.kt

```diff
@@ -130,8 +131,8 @@ fun ConversationMediaScreen(
     )
 
     SnackBarMessage(
-        messageComposerViewModel.infoMessage,
-        conversationMessagesViewModel.infoMessage
+        composerMessages = MutableSharedFlow<SnackBarMessage>().asSharedFlow(),
```

Comment:
shouldn't we be passing a shared flow variable here correctly so it can be listened to? this just seems like a new MutableSharedFlow but no one is emitting to it? 🤔

Reply:
It's used in 2 places where in one place it doesn't have second info message. I will try to figure better way to do it 💪

Reply:
maybe just make this parameter nullable 😄